### ISSUE TITLE
Change travis.yml to use `npm install --no-save`

### ISSUE THREAD

**Issue by matthewmorgan** [exercism/ecmascript#397]
As discovered in #390 by @tarunvelli, the main `package.json` file is getting rewritten by `npm install`, and causing the build to fail because the main file formatting no longer matches the `package.json` exercise files.

We can stop this from happening by explicitly specifying `npm install --no-save`, which prevents the `package.json` file from being altered at build.  This used to be the default, but with `npm` 5 and above it is not, which is why this mysteriously started breaking.

@exercism/ecmascripts is there any reason _not_ to make this change?  I can't think of a good reason why we would want the repository altered in some way that's not visible to the contributors during the build step, but please let me know if I am overlooking something.

implementing this is simple, as was demonstrated on #390.  

## travis.yml

```yml
---
language: node_js
node_js:
- 'node'
sudo: false
script:
- bin/fetch-configlet
- bin/configlet lint .
- make test-travis
install:
- npm install --no-save
```

The `install` block is the extent of the change.

**Comment by tarunvelli** [comment]
This seems perfectly reasonable as this is the expected behavior. I guess the same is applicable to https://github.com/exercism/javascript also.

**Comment by tejasbubane** [comment]
We definitely should have `--no-save` in CI - especially since we compare `package.json` files. 💯

**Comment by matthewmorgan** [comment]
Closed by #390
